<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT Windows Path Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="top">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="userGroupBox">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>120</height>
         </size>
        </property>
        <property name="title">
         <string>User Path:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="twUserPath">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEditUserPath">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btSelectUserPath">
               <property name="text">
                <string>Select</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btAddUserPath">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_add.png</normaloff>:/dir_add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btUpdUserPath">
             <property name="text">
              <string>Update</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_edit.png</normaloff>:/dir_edit.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btRemoveUserPath">
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_del.png</normaloff>:/dir_del.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btUpUserPath">
             <property name="text">
              <string>Up</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_up.png</normaloff>:/dir_up.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btDnUserPath">
             <property name="text">
              <string>Down</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_down.png</normaloff>:/dir_down.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="systemGroupBox">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>120</height>
         </size>
        </property>
        <property name="title">
         <string>System Path:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="twSystemPath">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEditSystemPath">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btSelectSystemPath">
               <property name="text">
                <string>Select</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btAddSystemPath">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_add.png</normaloff>:/dir_add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btUpdSystemPath">
             <property name="text">
              <string>Update</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_edit.png</normaloff>:/dir_edit.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btRemoveSystemPath">
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_del.png</normaloff>:/dir_del.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btUpSystemPath">
             <property name="text">
              <string>Up</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_up.png</normaloff>:/dir_up.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btDnSystemPath">
             <property name="text">
              <string>Down</string>
             </property>
             <property name="icon">
              <iconset resource="res/icons.qrc">
               <normaloff>:/dir_down.png</normaloff>:/dir_down.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
